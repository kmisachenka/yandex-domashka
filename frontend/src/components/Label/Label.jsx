import React from 'react';
import * as PropTypes from 'prop-types';

import styles from "./Label.module.scss";

export default function Label(props) {
  const { label } = props;
  return (
    <aside className={styles.label}>
      <h2 className={styles.title}>{label}</h2>
    </aside>
  );
}

Label.propTypes = {
  label: PropTypes.string.isRequired,
};
