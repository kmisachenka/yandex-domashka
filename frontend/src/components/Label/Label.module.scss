@import '../../media';

.label {
  align-items: center;
  display: flex;

  padding-bottom: 16px;

  @include tablet {
    padding-bottom: 32px;
    padding-top: 16px;
  }

  @include desktop {
    min-width: 1020px;
  }
}

.title {
  @include zero;

  display: block;
  font-family: Helvetica, serif;
  font-size: 36px;
  font-weight: normal;
  line-height: 42px;

  margin-right: 16px;
}
