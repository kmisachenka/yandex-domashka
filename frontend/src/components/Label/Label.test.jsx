import React from 'react';
import { shallow } from 'enzyme';

import Label from './Label';

describe('Label', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Label label="text" />);
    expect(wrapper).toMatchSnapshot();
  });
});
