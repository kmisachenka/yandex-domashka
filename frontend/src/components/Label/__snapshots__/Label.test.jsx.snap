// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should match snapshot 1`] = `
<aside
  className="label"
>
  <h2
    className="title"
  >
    text
  </h2>
</aside>
`;
